pkg/diagnostics/diagnostics.go

```diff
@@ -12,6 +13,12 @@ import (
 
 var logger = log.Logger("keep-diagnostics")
 
+// All available protocols https://github.com/multiformats/multiaddr#protocols
+// Eligible protocols are used to fetch the address to call the /diagnostics
+// endpoint
+var eligibleIP4 = "ip4"
+var eligibleDNS4 = "dns4"
```

Comment:
Are those used anywhere?

Reply:
Nah, I forgot to remove them after removing addresses filtering [414caae](https://github.com/keep-network/keep-core/pull/3255/commits/414caae76270fe8e549a361de8accf4d00fe9ba4)